lib/client.js

```diff
@@ -25,83 +51,69 @@ class Client {
     if (this.options.log_level) log.setLevel(this.options.log_level)
   }
 
-  get (path, opts) {
-    opts = opts || {}
-    opts = Object.assign({
-      cache: true,
-      authenticate: true,
-      token_expiration_retries: 1 // Number of refresh attempts to make if token expired (1 should do it!)
-    }, opts)
+  /**
+   * GET an api path
+   *
+   * @param {string} path - The path to fetch (e.g. 'current-schema/Item')
+   * @param {RequestOptions} options - A hash of options.
+   *
+   * @return {Promise} A promise that resolves the fetched data
+   */
+  get (path, options) {
+    options = Object.assign({
+      cache: true
```

Comment:
Since this is a breaking change, I wonder if we should also get rid of the default cache behavior. This has created a lot of confusion and I don't think caching should be enabled by default.

Reply:
Yeah we can change that default. I wasn't aware this created issues. What were the scenarios?

Reply:
The default has been changed. Also added a couple tests to confirm it works as expected https://github.com/NYPL-discovery/node-nypl-data-api-client/blob/c15d829f8be6a14e998217cf56a0c84e2faa87c5/test/cache-test.js